skysprites/lib/src/physics_shape.dart

```diff
@@ -53,6 +52,81 @@ class PhysicsShapePolygon extends PhysicsShape {
   }
 }
 
+class PhysicsShapeBox extends PhysicsShape {
+  PhysicsShapeBox(
+    this.width,
+    this.height, [
+      this.center = Point.origin,
+      this.rotation = 0.0
+  ]);
+
+  final double width;
+  final double height;
+  final Point center;
+  final double rotation;
+
+  box2d.Shape _createB2Shape(PhysicsNode node) {
+    box2d.PolygonShape shape = new box2d.PolygonShape();
+    shape.setAsBox(
+      width / node.b2WorldToNodeConversionFactor,
+      height / node.b2WorldToNodeConversionFactor,
+      new Vector2(
```

Comment:
If you're using the Google Vector2, there is an operator /(double scale) that you might like:

new Vector2(center.x, center.y) / node.b2WorldToNodeConversionFactor

But technically this does another memory allocation so if that bugs you, feel free to keep doing it this way :)